lib/account.php

```diff
@@ -69,18 +72,19 @@ public function getImapConnection() {
 			$host = $this->account->getInboundHost();
 			$user = $this->account->getInboundUser();
 			$password = $this->account->getInboundPassword();
+			$password = $this->crypto->decrypt($password);
```

Comment:
"as taken from the db" - you also call the function **before** you save the account in the database (at account creation - so only **once**), that's why it fails at this point. Later calls use a password from the database, which is encrypted.

So after the account is in the database (after the imap and smtp tests), your code works as expected.